docs/en/04_Changelogs/4.9.0.md

```diff
@@ -36,9 +49,10 @@ SilverStripe\Assets\Image:
   lazy_loading_enabled: false
 ```
 
-#### CMS authors
+#### How content authors can interact with image lazy loading
 
-CMS authors can disable lazy loading on images added via the WYSIWYG editor in the "Edit image" panel by setting the new "Loading" field to "Eager".
+Content authors can disable lazy loading on images added via the HTML editor field in the 
+***Insert media*** dialog by setting the new ***Loading*** field to ***Eager***.
```

Comment:
Need to add a link to the user help doc here

Reply:
I don't think we need to say 'new', as this text will end up being in there for years to come.